aws/resource_aws_eks_cluster_test.go

```diff
@@ -424,6 +424,43 @@ func TestAccAWSEksCluster_VpcConfig_PublicAccessCidrs(t *testing.T) {
 	})
 }
 
+func TestAccAWSEksCluster_NetworkConfig_ServiceIpv4Cidr(t *testing.T) {
```

Comment:
if you don't mind, can you add a test for the case where the empty config block (`kubernetes_network_config {}`) is provided to test the case where AWS supplies the IP?

Reply:
Good shout, have added this to the _Basic test